test/command/suite/io_flush/recursive/dependent/data_column/create.test

```diff
@@ -0,0 +1,11 @@
+table_create Users TABLE_HASH_KEY ShortText
+
+#@collect-query-log true
+io_flush --recursive dependent
+#@collect-query-log false
```

Comment:
What is tested here?

Reply:
@kou 
It is intended to compare flushed result before/after creating table.
Is this `create.test` itself should be removed?

Reply:
I don't know why we should compare flushed result before/after creating table.
So I think that this file is needless.